src/modules/seventv/channel-emotes.js

```diff
@@ -89,9 +91,10 @@ class SevenTVChannelEmotes extends AbstractEmotes {
     let message;
     switch (action) {
       case 'ADD': {
-        if (isUnlisted(emote.visibility)) {
-          return;
-        }
+        const hideUnlisted =
+          hasFlag(settings.get(SettingIds.EMOTES), EmoteTypeFlags.SEVENTV_HIDE_UNLISTED) &&
+          isUnlisted(emote.visibility);
```

Comment:
this seems to filter unlisted and permanently unlisted emotes. what is the difference between unlisted and permanently unlisted?

Reply:
No idea, can't find anything about that in their code or on website.

Reply:
iirc: `unlisted` = hasn't been vetted/verified by 7TV moderators
permanently unlisted = vetted by 7TV moderators to not be shown on public listings, can still be added to channels, but can only be accessed via a link.